src/views/Tutorials.vue

```diff
@@ -0,0 +1,170 @@
+                    <h3 id="tutorial-node-connect">Connect your GRIN-Node to Grinnode.live&nbsp;</h3>
+                      <p><router-link exact to="/faq">1. Setup a GRIN Node</router-link> </p>
+                      <p>2. Edit your Grin-Node <strong>grin-server.toml</strong> </p>
+                          <pre>
+                              <code>
+  peers_preferred = ["213.239.215.236:3414"]
+
+  #maximum number of inbound peer connections
+  peer_max_inbound_count = 30
+
+  #maximum number of outbound peer connections
+  peer_max_outbound_count = 10
+
+  #preferred minimum number of outbound peers
+  peer_min_preferred_outbound_count = 10
+
+  #temporarily allowed to exceed peer_max_inbound_count
+  peer_listener_buffer_count = 5
+                             </code>
+                          </pre>
+                  <h3 id="tutorial-seed">Use Grinnode.live High-Availale Seed-Node </h3>
+                  <p> Connect to lot of Grin-Nodes fast using our Seed-Node by editing your grin-server.toml.
+                      Or download our example <a href="https://github.com/MCM-Mike/grinnode.live/blob/master/grin-server.toml">grin-server.toml</a>
+                  </p>
+                      <pre>
+                              <code>
+  #how to seed this server, can be None, List or DNSSeed
+  seeding_type = "List"
+
+  #If the seeding type is List, the list of peers to connect to can
+  #be specified as follows:
+  seeds = ["mainnet-seed.grinnode.live:3414"]
+
+                              </code>
+                      </pre>
+                  <h3 id="tutorial-wallet-connect">Connect your GRIN-Wallet to Grinnode.live&nbsp;</h3>
+                      <p>1. Setup a GRIN Wallet </p>
+                      <p>2. Use it as <strong><a href="https://github.com/mimblewimble/docs/wiki/How-to-use-the-Grin-wallet#connecting-to-a-node" target="_blank" rel="noopener">API Endpoint</a></strong> or <strong>add</strong> it to your <a href="https://github.com/mimblewimble/docs/wiki/How-to-use-the-Grin-wallet#connecting-to-a-node"><strong>grin-wallet.toml</strong>&nbsp;</a>
+                      </p>
+                      <blockquote>
+                          <p><strong>CLI:<br /></strong>&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp;./grin-wallet --api_server_address "https://grinnode.live:3413"<br /><br /><strong>grin-wallet.toml</strong>&nbsp;:<br />&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; check_node_api_http_addr = "https://grinnode.live:3413"</p>
+                          <p>&nbsp;</p>
+                      </blockquote>
+                      <p> More Grin-Wallet example commands can be found <a href="https://github.com/MCM-Mike/grinnode.live/blob/master/linux-grin-commands-collection.md">here</a></p>
+                      <p> When using the Grinnode Wallet API , you are trusting us to run a Grin node. It is recommended to run your own Grin-Node and use it instead of our API which is better for your privacy.</p>
```

Comment:
`It is recommended to run your own Grin-Node and use it instead of our API which is better for your privacy.` --> We recommend to use your own node for better privacy.

Reply:
Updated! Thanks!